superbench/benchmarks/micro_benchmarks/dist_inference_cpp/dist_inference.cu

```diff
@@ -45,6 +45,13 @@
 #include <hipblaslt/hipblaslt.h>
 #include <rccl/rccl.h>
 using cublasLtHalf = hipblasLtHalf;
+#if defined(HIPBLASLT_DATATYPE_WORKAROUND)
```

Comment:
@yukirora please check whether we should find a version to switch it.

Reply:
Fixed naming.